### PR TITLE
support for Flaky, Epic, Feature, Issue, Link, Muted, Owner, Severity…

### PR DESCRIPTION
**Pull request by yuri-kudryavtsev**
Added support for Flaky, Epic, Feature, Issue, Link, Muted, Owner, Severity, Story, TmsLink annotations
Added feature combinations test
Added missing aoc-ajc.xml file

### CODE REVIEW

File: allure-spock/src/main/java/io/qameta/allure/spock/AllureSpock.java [hunk 1]
```diff
@@ -1,12 +1,8 @@
 package io.qameta.allure.spock;
 
-import io.qameta.allure.Allure;
-import io.qameta.allure.AllureLifecycle;
-import io.qameta.allure.model.Label;
-import io.qameta.allure.model.Parameter;
-import io.qameta.allure.model.Status;
-import io.qameta.allure.model.StatusDetails;
-import io.qameta.allure.model.TestResult;
+import io.qameta.allure.*;
```
Comment (baev) [inline allure-spock/src/main/java/io/qameta/allure/spock/AllureSpock.java]: do not use wildcard imports

File: allure-spock/src/test/groovy/io/qameta/allure/spock/FeatureCombinationsTest.groovy [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+@Feature("Feature annotation")
+@Title("Title spec annotation")
+@Link("https://www.yandex.ru/")
+@Issue("AKiOS-280")
+@TmsLink("TEST1")
+@Flaky
+@Muted
+@Epic("EPIC LABEL!")
+@Owner("yuri.kudryavtsev")
+class FeatureCombinationsTest extends Specification {
```
Comment (baev) [inline allure-spock/src/test/groovy/io/qameta/allure/spock/FeatureCombinationsTest.groovy]: you need to write test that  runs spock tests and verifies stored data

**Comment by eroshenkoam** [comment]
LGTM

**Comment by baev** [comment]
@yuri-kudryavtsev thanks!

**Comment by eroshenkoam** [comment]
@yuri-kudryavtsev 🚀

**Pull request merged**